tempesta_fw/http.h

```diff
@@ -83,6 +83,7 @@ enum {
 #define TFW_HTTP_PF_CR			0x01
 #define TFW_HTTP_PF_LF			0x02
 #define TFW_HTTP_PF_CRLF		(TFW_HTTP_PF_CR | TFW_HTTP_PF_LF)
+#define TFW_HTTP_URI_FULL		0x04
```

Comment:
Wrong place. The block of definitions is for `parser->flags`, while you're using it in `req->flags`